rules/linux/builtin/lnx_crontab_file_modification.yml

```diff
@@ -0,0 +1,22 @@
+title: Modifying Crontab With File
```

Comment:
Are you sure that `REPLACE` appear with File only ?
From internet search, it's seen that  `REPLACE` is use when a tash is edited and update.

Reply:
Hi, been testing this more, and yes you are correct, will amend this to any change then.